### DISCUSSION TITLE
Organising the traits in tabs like the styles are organised

### DISCUSSION THREAD

**Discussion by anicauli** [GrapesJS/grapesjs#4941] [Q&A]
Hello,

I have many traits to include to my editor and I want to have them organised.

The way how I wish to do it is like the styles are organised from the build in functionalities.
![image](https://user-images.githubusercontent.com/68219083/220693517-e509e95e-4130-4a54-875d-6579bd31dff9.png)

Is there a possibility to do it also for the traits?

Thanks!

**Comment by artf** [comment]
Not by default, unfortunately.

**Reply by anicauli** [reply]
Is there any way how I can do it? Maybe in programmatic way?
